src/main/java/uk/gov/justice/digital/delius/controller/secure/StaffResource.java

```diff
@@ -99,4 +121,65 @@ public StaffDetails getStaffDetailsForUsername(
         @PageableDefault(sort = {"secondName", "firstName"}, direction = Direction.ASC, size = Integer.MAX_VALUE) final Pageable pageable) {
         return staffService.getManageSupervisionsEligibleOffendersByUsername(username, pageable);
     }
+
+    @ApiOperation(
+        value = "Return the full caseload for a probation staff/officer",
+        notes = "Currently, this endpoint is restricted to offender managers and order supervisors. Additional management types (e.g. requirements, reports) may be added later.")
+    @GetMapping(path = "/staff/staffIdentifier/{staffIdentifier}/caseload")
+    public Caseload getCaseloadForStaff(
+        @ApiParam(name = "staffIdentifier", value = "Delius staff/officer identifier", example = "123456", required = true)
+        @NotNull @PathVariable(value = "staffIdentifier") final Long staffIdentifier) {
+        return caseloadService.getCaseloadByStaffIdentifier(staffIdentifier, OFFENDER_MANAGER, ORDER_SUPERVISOR)
+            .orElseThrow(() -> new NotFoundException(String.format("Staff member with identifier %d", staffIdentifier)));
+    }
+
+    @ApiOperation(value = "Return the full caseload for a probation staff/officer, returning only the managed offenders")
+    @GetMapping(path = "/staff/staffIdentifier/{staffIdentifier}/caseload/managedOffenders")
+    public Set<ManagedOffenderCrn> getCaseloadOffendersForStaff(
+        @ApiParam(name = "staffIdentifier", value = "Delius staff/officer identifier", example = "123456", required = true)
+        @NotNull @PathVariable(value = "staffIdentifier") final Long staffIdentifier) {
+        return caseloadService.getCaseloadByStaffIdentifier(staffIdentifier, OFFENDER_MANAGER)
+            .map(Caseload::getManagedOffenders)
+            .orElseThrow(() -> new NotFoundException(String.format("Staff member with identifier %d", staffIdentifier)));
+    }
+
+    @ApiOperation(value = "Return the full caseload for a probation staff/officer, returning only the supervised orders/events")
+    @GetMapping(path = "/staff/staffIdentifier/{staffIdentifier}/caseload/supervisedOrders")
+    public Set<ManagedEventId> getCaseloadOrdersForStaff(
+        @ApiParam(name = "staffIdentifier", value = "Delius staff/officer identifier", example = "123456", required = true)
+        @NotNull @PathVariable(value = "staffIdentifier") final Long staffIdentifier) {
+        return caseloadService.getCaseloadByStaffIdentifier(staffIdentifier, ORDER_SUPERVISOR)
+            .map(Caseload::getSupervisedOrders)
+            .orElseThrow(() -> new NotFoundException(String.format("Staff member with identifier %d", staffIdentifier)));
+    }
```

Comment:
I'm not sure of the differences between a staff `caseload`, their `managedOffenders` and their `supervisedOrders` ? Could you expand a little on those differences please?

Reply:
Sure - basically a staff member's caseload consists of all the "things" they are responsible for in Delius, which can include offenders, orders (aka events), licence conditions, post-sentence supervision requirements etc etc.  

The `caseload` endpoint returns everything*, whereas the `caseload/managedOffenders` and `caseload/supervisedOrders` endpoints are filtered down to the offender-level and order/event-level management respectively.  As I understand it you only need the offender-level management details at this point, so the `caseload/managedOffenders` endpoint should be what you're after.

*The `caseload` endpoint doesn't technically return everything - only the offender-level and event-level management.